snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: ktor # you probably want to 'snapcraft register <name>'
+summary: Creates porjects with ktor framework setup.
+version: '0.0.1' # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
should we align the version with Ktor and use 2.1.0-eap-1?

Reply:
Will do, but for now this is a very first temporary release which I'm planning to remove later once the publication is set up properly